src/runtime/runtime_stm32wle5.go

```diff
@@ -4,21 +4,125 @@
 package runtime
 
 import (
+	"device/stm32"
 	"machine"
 )
 
 type arrtype = uint32
 
 func init() {
-	// Currently the clock is not configured, which means
-	// the MCU runs at default reset clock speed (4Mhz).
-	// Code to initialize RCC and PLL can go here.
+
+	// Configure 48Mhz clock
+	initCLK()
 
 	// UART init
 	machine.Serial.Configure(machine.UARTConfig{})
 
 	// Timers init
 	initTickTimer(&machine.TIM1)
+
+	// Radio init
+	subGhzInit()
+}
+
+func initCLK() {
+
+	// Set Power Voltage Regulator Range 2
+	//stm32.PWR.CR1.ReplaceBits(0b10, stm32.PWR_CR1_VOS_Msk, stm32.PWR_CR1_VOS_Pos)
+
+	// Enable HSE32 VDDTCXO output on package pin PB0-VDDTCXO
+	stm32.RCC.CR.ReplaceBits(stm32.RCC_CR_HSEBYPPWR_VDDTCXO, 0b1, stm32.RCC_CR_HSEBYPPWR_Pos)
+
+	// SYSCLOCK from HSE32 clock division factor (SYSCLOCK=HSE32)
+	stm32.RCC.CR.ReplaceBits(stm32.RCC_CR_HSEPRE_Div1, 0b1, stm32.RCC_CR_HSEPRE_Pos)
+
+	// enable external Clock HSE32 TXCO (RM0461p226)
+	stm32.RCC.CR.SetBits(stm32.RCC_CR_HSEBYPPWR)
+	stm32.RCC.CR.SetBits(stm32.RCC_CR_HSEON)
+	for !stm32.RCC.CR.HasBits(stm32.RCC_CR_HSERDY) {
+	}
+
+	// Disable PLL
+	stm32.RCC.CR.ClearBits(stm32.RCC_CR_PLLON)
+	for stm32.RCC.CR.HasBits(stm32.RCC_CR_PLLRDY) {
+	}
+
+	// Configure PLL
+	stm32.RCC.PLLCFGR.Set(0x23020613)
```

Comment:
Can use symbolic constants here?  E.g. other STM32 worked on recently tend to have something like this:
```
	// Configure the PLL
	stm32.RCC.PLLCFGR.ReplaceBits(
		(1)| // 1 = RCC_PLLSOURCE_MSI
			(PLL_M-1)<<stm32.RCC_PLLCFGR_PLLM_Pos|
			(PLL_N<<stm32.RCC_PLLCFGR_PLLN_Pos)|
			(((PLL_Q>>1)-1)<<stm32.RCC_PLLCFGR_PLLQ_Pos)|
			(((PLL_R>>1)-1)<<stm32.RCC_PLLCFGR_PLLR_Pos)|
			(PLL_P<<stm32.RCC_PLLCFGR_PLLPDIV_Pos),
		stm32.RCC_PLLCFGR_PLLSRC_Msk|stm32.RCC_PLLCFGR_PLLM_Msk|
			stm32.RCC_PLLCFGR_PLLN_Msk|stm32.RCC_PLLCFGR_PLLP_Msk|
			stm32.RCC_PLLCFGR_PLLR_Msk|stm32.RCC_PLLCFGR_PLLPDIV_Msk,
		0)
```

Reply:
I could rewrite the PLL init with symbolic constants . Done